aip/auth/4111.md

```diff
@@ -27,7 +27,7 @@ JWT as a means of authentication.
 ### Application Default Credentials
 
 Supporting self-signed JWT is considered a part of [application default
-credentials][2]. To understand the overall flow, please read [AIP-4110][2].
+credentials][2]. To understand the overall flow, please read [AIP-4110][0].
```

Comment:
Thank you for fixing this! Can you update the [2] to [0] after application default credentials as well? Thanks!